### PR TITLE
Aft2

### PR DESCRIPTION
**Pull request by pirog**
Thank you so much for contributing code to Lando! If this is your **first time** contributing to Lando we recommend you first check our [Getting Involved](https://docs.lando.dev/contrib/contributing.html) docs.

It may also be useful to check the below documentation based on what you are trying to do:

* [Adding code to Lando](https://docs.lando.dev/contrib/contrib-intro.html)
* [Updating documentation](https://docs.lando.dev/contrib/contrib-intro.html)
* [Working on DevOps](https://docs.lando.dev/contrib/contrib-intro.html)
* [Working on Lando's auxiliary services eg websites, apis, etc](https://docs.lando.dev/contrib/contrib-intro.html)
* [Writing Lando Guides](https://docs.lando.dev/contrib/guides-intro.html)
* [Writing Lando blog posts](https://docs.lando.dev/contrib/blogging-intro.html)

We will get to your PR as soon as we can but in the meantime you might want to check to make sure you have done the following. **The more boxes you can check the easier it will be for us to merge in your changes with confidence**

- [ ] My code includes the latest code from the `master` branch
- [ ] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/help)
- [ ] My code includes a [functional test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes a [unit test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: plugins/lando-services/services/memcached/launch.sh [hunk 1]
```diff
@@ -4,4 +4,6 @@
 set -e
 
 # Try the new entrypoint and then fallback to the older one
-/entrypoint.sh /run.sh || /app-entrypoint.sh /run.sh
+/opt/bitnami/scripts/memcached/entrypoint.sh /opt/bitnami/scripts/memcached/run.s \
```
Comment (uberhacker) [inline plugins/lando-services/services/memcached/launch.sh]: Change `run.s` to `run.sh`?

File: plugins/lando-services/services/mongo/launch.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Errors and logz
+set -e
+
+# Try the new entrypoint and then fallback to the older one
+/opt/bitnami/scripts/memcached/entrypoint.sh /opt/bitnami/scripts/memcached/run.s \
```
Comment (uberhacker) [inline plugins/lando-services/services/mongo/launch.sh]: Change `run.s` to `run.sh`?

File: plugins/lando-services/services/elasticsearch/builder.js [hunk 3]
```diff
@@ -25,7 +29,7 @@ module.exports = {
       options = _.merge({}, config, options);
       const elasticsearch = {
         image: `bitnami/elasticsearch:${options.version}`,
-        command: '/entrypoint.sh /run.sh',
+        command: '/bin/sh -c "chmod +x /launch.sh && /launch.sh"',
```
Comment (uberhacker) [inline plugins/lando-services/services/elasticsearch/builder.js]: Why not simply `command: 'chmod +x /launch.sh && /launch.sh',`?

File: plugins/lando-services/services/mongo/builder.js [hunk 4]
```diff
@@ -23,13 +28,14 @@ module.exports = {
       options = _.merge({}, config, options);
       const mongo = {
         image: `bitnami/mongodb:${options.version}`,
-        command: '/entrypoint.sh /run.sh',
+        command: '/bin/sh -c "chmod +x /launch.sh && /launch.sh"',
```
Comment (uberhacker) [inline plugins/lando-services/services/mongo/builder.js]: Not understanding the need to wrap in `/bin/sh -c "..."`.

File: plugins/lando-services/services/memcached/launch.sh (hunk 1 above)
Comment (pirog) [reply plugins/lando-services/services/memcached/launch.sh]: good catch

File: plugins/lando-services/services/mongo/launch.sh (hunk 2 above)
Comment (pirog) [reply plugins/lando-services/services/mongo/launch.sh]: good catch

**Pull request merged**